include/pybind11/pybind11.h

```diff
@@ -69,9 +69,14 @@ inline bool apply_exception_translators(std::forward_list<ExceptionTranslator> &
     return false;
 }
 
+constexpr const char* function_capsule_name = "pybind11_funciton_capsule_v1";
```

Comment:
There is a typo. And maybe this should be colocated with the function_record struct with a comment to update when relevant.